Documentation/building/unix-instructions.md

```diff
@@ -21,50 +95,33 @@ cd corefx
 ```bash
 managed            # optional argument to build the managed code
 native             # optional argument to build the native code
+platform           # OS to compile for (FreeBSD, Linux, NetBSD, OSX, Windows)
+skiptests          # build, but do not run, the managed unit tests
+BuildType          # build configuration type (Debug, Release)
 
 # The following arguments affect native builds only:
 
 BuildArch          # build architecture (x64, x86, arm, arm64)
-BuildType          # build configuration type (Debug, Release)
 clean              # optional argument to force a clean build
 verbose            # optional argument to enable verbose build output
 clangx.y           # optional argument to build using clang version x.y
-platform           # OS to compile for (FreeBSD, Linux, NetBSD, OSX, Windows)
 cross              # optional argument to signify cross compilation, uses ROOTFS_DIR environment variable if set
-skiptests          # skip the tests in the './bin/*/*Tests/' subdirectory
 cmakeargs          # user-settable additional arguments passed to CMake
-```
-
-### Prerequisites
-
-* git
-* curl-dev (libcurl4-openssl-dev)
-* icu (libicu52)
-* cmake
-* clang (clang-3.5)
-* libunwind (libunwind8)
-
-> Note: These instructions have been validated on:
-* Ubuntu 15.04, 14.04, and 12.04
-* Fedora 22
-* OS X 10.10 (Yosemite)
-* FreeBSD 10.2
-* NetBSD 7.0
-* Alpine Linux 3.3
 
+```
 ### Known Issues
 If you see errors along the lines of `SendFailure (Error writing headers)` you may need to import trusted root certificates:
 
 ```sh
 mozroots --import --sync
 ```
 
-## Steps to Build on Ubuntu
+## Steps to build on Ubuntu 14.04 LTS
```

Comment:
> Perhaps future work, but in this particular case it looks :+1: to me! :smile:
 
We are working on producing .NET Core CLI for Ubuntu 16.04 (ideally done this week).  Once that's complete, I plan to flush this out for other platforms.  Right now, It's not really possible to do a managed build on Ubuntu 16.04 without all manner of hacks.  I'd rather wait until the managed tool-chain works there and then update it the right way.